README.md

```diff
@@ -25,27 +25,23 @@ dependencies {
 ```
 
 - Know your tenant (BIDTenant) `dns` and `communityName`
-- Initialize SDK
-```
-BIDSDK.getInstance().setupTenant(<your tenant>, "<your license key>");
-```
 
 - Request OTP
 ```
-BIDOtpResponse otpResponse = BIDOTP.requestOTP("<username>", "<emailTo>", "<smsTo>", "1");
+BIDOtpResponse otpResponse = BIDOTP.requestOTP({ "dns": "<dns>", "communityName": "<communityName>", "licenseKey": "<licenseKey>" },"<username>", "<emailTo>", "<smsTo>", "1");
```

Comment:
@jenish4096 this appears to be JS code structure.. can we please fix this.

Reply:
@manik-1kosmos can we do like below?

```
Map<String, Object> tenantInfo = new HashMap<>();
tenantInfo.put("dns", <dns>);
tenantInfo.put("communityName", <communityName>);
tenantInfo.put("licenseKey", <licenseKey>);

BIDOtpResponse otpResponse = BIDOTP.requestOTP(tenantInfo, "<username>", "<emailTo>", "<smsTo>", "1");
```